databases/databases/databases_and_sql.md

```diff
@@ -18,52 +18,17 @@ SQL is one of those topics that's been stored away in dusty old technical manual
 
 Though the prevalence of web applications these days has grown the demand among new users to focus on understanding the *concepts* of SQL, the learning tools haven't really caught up. We'll do our best to impart those concepts using the tools available.
 
-### Learning Outcomes
-Look through these now and then use them to test yourself after doing the assignment:
-
-#### Important Concepts:
-
-* What is the Primary Key?
-* What are Foreign Keys?
-* What is a Schema?
-
-#### Statements:
-* `SELECT`
-* `CREATE TABLE`
-* `DROP TABLE`
-* `CREATE INDEX`
-* `DROP INDEX`
-* `UPDATE`
-* `DELETE`
-* `INSERT INTO`
-* `CREATE DATABASE`
-* `DROP DATABASE`
-* `COMMIT` (concept)
-* `ROLLBACK` (concept)
-
-#### Clauses:
-* `DISTINCT`
-* `WHERE`
-* `IN`
-* `AND`
-* `OR`
-* `BETWEEN`
-* `LIKE`
-* `ORDER BY`
-* `COUNT`
-
-#### Functions:
-* `GROUP BY`
-* `HAVING`
-* `AVG`
-* `COUNT`
-* `MIN`
-* `MAX`
-* `SUM`
-
-#### Other Stuff:
-* What are Indexes good for?
-* What's the difference between `WHERE` and `HAVING`?
+### Lesson Overview
+This section contains a general overview of topics that you will learn in this lesson.
+
+- What the Primary Key is.
```

Comment:
I think "What a Primary Key is" reads a bit more naturally. Thoughts?

Reply:
True. Thanks for noting!